src/test/java/plugins/LdapPluginTest.java

```diff
@@ -73,6 +90,24 @@ public void login_ok() {
     }
 
     /**
+     * Scenario: Login with ldap uid and password
+     * Given I have a docker fixture "ldap" that allows anonymous binding
+     * And Jenkins is using ldap as security realm
+     * When I login with user "jenkins" and password "root" without manager credentials
+     * Then I will be successfully logged on as user "jenkins"
```

Comment:
My bad @olivergondza . Read the comment to quickly and I misunderstood. If the convention is not to add the comments, I'll remove them. I'm of the opinion that comments are always good though :)